service/src/main/java/aero/minova/cas/controller/FilesController.java

```diff
@@ -37,13 +37,24 @@
 
 import aero.minova.cas.CustomLogger;
```

Comment:
Bitte das Ganze kurz zusammengefasst noch in die CHANGELOG.md eintragen. Links kann man da auch nutzen.